src/login/refresh-token.jsx

```diff
@@ -0,0 +1,11 @@
+const refreshToken = async(handle, refreshTiming) => {
+  const newAuthRes = await handle.reloadAuthResponse()
+  refreshTiming = (newAuthRes.expires_in || 3600 - 5 * 60) * 1000
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/126044620-a27507b4-3e04-4295-b4d2-3786542cfc47.png)
Me seeing random math not extracted in its own separate function